alicloud/extension_drds.go

```diff
@@ -0,0 +1,13 @@
+package alicloud
+
+type InstanceType string
+type DRDSInstancePayType string
+
+const (
+	PrivateType             = InstanceType("PRIVATE")
+	PublicType              = InstanceType("PUBLIC")
+	PrivateType_            = InstanceType("1")
+	PublicType_             = InstanceType("0")
```

Comment:
I think `PrivateType_` and `PublicType_` are redundant and just leave  `PrivateType` and `PublicType` is ok.

Reply:
fix